src/MIDebugEngine/AD7.Impl/AD7Engine.cs

```diff
@@ -569,9 +569,8 @@ public int CauseBreak()
         // and the debugger does not want to actually enter break mode.
         public int Continue(IDebugThread2 pThread)
         {
-            AD7Thread thread = (AD7Thread)pThread;
-
-            _pollThread.RunOperation(() => _debuggedProcess.Continue(thread.GetDebuggedThread()));
+            AD7Thread thread = pThread as AD7Thread;
```

Comment:
Might want to add a note here indicating why we are doing this. Something like:
// VS Code currently has a bug where a thread isn't provided in some cases. Work
// around this by passing zero as the thread for now.